stdlib/public/core/CompilerProtocols.swift

```diff
@@ -338,7 +338,7 @@ public protocol ExpressibleByUnicodeScalarLiteral {
   /// A type that represents a Unicode scalar literal.
   ///
   /// Valid types for `UnicodeScalarLiteralType` are `UnicodeScalar`,
-  /// `String`, and `StaticString`.
+  /// `Character`, `String`, and `StaticString`.
```

Comment:
Should be addressed by https://github.com/apple/swift/pull/7125/files#diff-4d7f8659b54e15ede9391722ae11cb74R24

Was this what you had in mind?

I‘m just verifying the tests locally, then I'll ping you so we can run them with CI and finalise the PR.